test-data/unit/check-expressions.test

```diff
@@ -2803,3 +2803,11 @@ def f() -> int:  # E: Missing return statement
     x: int
     assert isinstance(x, int), '...'
 [builtins fixtures/isinstance.pyi]
+
+[case testTypeVarAsValue]
+from typing import TypeVar
+T = TypeVar("T")
+x: int
+x + T  # E: Unsupported left operand type for + ("int")
```

Comment:
Hm, isn't this a little weird? I'd think that the right operand type is more suspect... :-)

Reply:
Good spot, it is a little weird. It took me some pdb, but the difference is that `int` in [builtins fixtures/tuple.pyi] doesn't have `__add__`. The fixture is actually unnecessary, so it's gone now :-)